### ISSUE TITLE
Unresolved reference: Issue (script.how-to-fix-text-provider.kts:21:41)

### ISSUE THREAD

**Issue by derwehr** [oss-review-toolkit/ort-ci-github-action#17]
Without changing the yaml file (am running the default script) or any ort configs (I don't have any), the ort-reporter stopped working, and throws the error message below.
(Could this be related to [renaming OrtIssue](https://github.com/oss-review-toolkit/ort/commit/6b292bdb21dc9bc4f8cb18fa7db6d0a804adf0ff) from last week?)

```
________ _____________________
\_____  \\______   \__    ___/ the OSS Review Toolkit, version DOCKER-SNAPSHOT.
 /   |   \|       _/ |    |
/    |    \    |   \ |    |    Running 'report' as '?' under Java 11.0.17 on Linux
\_______  /____|_  / |____|    with 2 CPUs and a maximum of 5120 MiB of memory.
        \/       \/
Environment variables:
ORT_CONFIG_DIR = /home/ort/.ort/config
ORT_DATA_DIR = /home/ort/.ort
JAVA_HOME = /opt/java/openjdk
ANDROID_HOME = /opt/android-sdk

Looking for ORT configuration in the following file:
	/home/ort/.ort/config/config.yml (does not exist)

10:28:14.517 [main] ERROR org.ossreviewtoolkit.utils.scripting.ScriptRunner - Unresolved reference: Issue (script.how-to-fix-text-provider.kts:21:41)
Exception in thread "main" java.lang.RuntimeException: Using JVM IR backend
Using JVM IR backend
Loading modules: [java.se, jdk.accessibility, jdk.attach, jdk.compiler, jdk.dynalink, jdk.httpserver, jdk.jartool, jdk.javadoc, jdk.jconsole, jdk.jdi, jdk.jfr, jdk.jshell, jdk.jsobject, jdk.management, jdk.management.jfr, jdk.naming.ldap, jdk.net, jdk.scripting.nashorn, jdk.sctp, jdk.security.auth, jdk.security.jgss, jdk.unsupported, jdk.unsupported.desktop, jdk.xml.dom, java.base, java.compiler, java.datatransfer, java.desktop, java.xml, java.instrument, java.logging, java.management, java.management.rmi, java.rmi, java.naming, java.net.http, java.prefs, java.scripting, java.security.jgss, java.security.sasl, java.sql, java.transaction.xa, java.sql.rowset, java.xml.crypto, jdk.internal.jvmstat, jdk.management.agent, jdk.jdwp.agent, jdk.internal.ed, jdk.internal.le, jdk.internal.opt]
Loading modules: [java.se, jdk.accessibility, jdk.attach, jdk.compiler, jdk.dynalink, jdk.httpserver, jdk.jartool, jdk.javadoc, jdk.jconsole, jdk.jdi, jdk.jfr, jdk.jshell, jdk.jsobject, jdk.management, jdk.management.jfr, jdk.naming.ldap, jdk.net, jdk.scripting.nashorn, jdk.sctp, jdk.security.auth, jdk.security.jgss, jdk.unsupported, jdk.unsupported.desktop, jdk.xml.dom, java.base, java.compiler, java.datatransfer, java.desktop, java.xml, java.instrument, java.logging, java.management, java.management.rmi, java.rmi, java.naming, java.net.http, java.prefs, java.scripting, java.security.jgss, java.security.sasl, java.sql, java.transaction.xa, java.sql.rowset, java.xml.crypto, jdk.internal.jvmstat, jdk.management.agent, jdk.jdwp.agent, jdk.internal.ed, jdk.internal.le, jdk.internal.opt]
Unresolved reference: Issue
	at kotlin.script.experimental.api.ErrorHandlingKt.valueOrThrow(errorHandling.kt:305)
	at org.ossreviewtoolkit.utils.scripting.ScriptRunner.runScript(ScriptRunner.kt:78)
	at org.ossreviewtoolkit.reporter.HowToFixScriptRunner.run(HowToFixTextProvider.kt:65)
	at org.ossreviewtoolkit.reporter.HowToFixTextProvider$Companion.fromKotlinScript(HowToFixTextProvider.kt:46)
	at org.ossreviewtoolkit.cli.commands.ReporterCommand.run(ReporterCommand.kt:243)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:198)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:211)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:18)
	at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:400)
	at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:397)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:415)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:440)
	at org.ossreviewtoolkit.cli.OrtMainKt.main(OrtMain.kt:82)
Error: Process completed with exit code 1.
```

**Comment by sschuberth** [comment]
> Could this be related to [renaming OrtIssue](https://github.com/oss-review-toolkit/ort/commit/6b292bdb21dc9bc4f8cb18fa7db6d0a804adf0ff) from last week?

Yes 😁 The version of ORT used by the action needs to be updated. PRs welcome 😉

**Comment by derwehr** [comment]
This was fixed in e238c9f7ee62beba523451bc8860c85f4c1a602d
